electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -40,7 +40,49 @@ def to_list(self) -> List[Dict[str, Any]]:
         return [event.to_dict() for event in self.events]
 
 
-class ExchangeList(EventList):
+class MergeableList(EventList, ABC):
```

Comment:
The name is not very self explanatory.

Reply:
Any suggestions, nothing better comes to mind.
To me it describes best what this type of list is capable.

Reply:
```suggestion
class MergeableEventList(EventList, ABC):
```
? But sounds like a repetion since Event is alreedy contained in the parent class.

Reply:
My vote is for MergeableEventList. It creates some repetition but I think it's easier to understand. And if we add """comments""" IDEs should make it easy enough to understand?